tests/TestCase/Datasource/ConnectionTest.php

```diff
@@ -36,20 +35,6 @@ public function tearDown(): void
         Log::drop('elasticsearch');
     }
 
-    /**
-     * Tests the getIndex method that calling it with no arguments,
-     * which is not supported
-     *
-     * @return void
-     */
-    public function testGetEmptyIndex()
-    {
-        $this->expectException(InvalidException::class);
-
-        $connection = new Connection();
-        $connection->getIndex();
-    }
-
```

Comment:
I removed that test because I was not able to reproduce that `InvalidException` from Elastica no matter what I tried.